modules/services/sharedtest/src/main/java/au/com/shiftyjelly/pocketcasts/sharedtest/FakeCrashLogging.kt

```diff
@@ -0,0 +1,27 @@
+package au.com.shiftyjelly.pocketcasts.sharedtest
+
+import com.automattic.android.tracks.crashlogging.CrashLogging
+import com.automattic.android.tracks.crashlogging.JsException
+import com.automattic.android.tracks.crashlogging.JsExceptionCallback
+
+class FakeCrashLogging : CrashLogging {
+    override fun initialize() {
+        TODO("Not yet implemented")
+    }
+
+    override fun recordEvent(message: String, category: String?) {
+        TODO("Not yet implemented")
+    }
```

Comment:
I'm not sure you can do that here. These things can happen in test scenarios so probably it should be implemented with `= Unit` or some other no-op indication.

Other `TODO`s are probably fine.

Reply:
Thanks, good point! I went ahead and added `Unit` to all methods - we really likely don't want to crash tests for "unimplemented fake crash logging".